.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "parser": "babel-eslint",
+    "extends": [
+      "airbnb-base",
+      "plugin:react/recommended"
+    ],
+    "settings": {
+        "version": "detect"
+    },
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "arrow-parens": "off"
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
```

Comment:
- [ ] This version of the .eslintrc.json file is different from the one provided by Microverse [here](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json), for the React module. We need to update this file.

- [ ] I also noticed that most Eslint dependencies were not added to your repository. This will make Eslint to not work properly with the default configuration provided by Microverse. Check the instructions [here](https://github.com/microverseinc/linters-config/tree/master/react-redux) on how to to set Eslint locally.